release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -126,6 +126,15 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
 
 //For more information, see < insert link to Deploying OpenStack with rootVolume and etcd on local disk when docs merge>.
 
+[id="ocp-4.15-postinstallation-configuration"]
+=== Postinstallation configuration 
+
+[id="ocp-4.15-postinstallation-configuration-multi-arch-compute-machines"]
+
+==== {product-title} clusters with multi-architecture compute machines
+
+On {product-title} {product-version} clusters with multi-architecture compute machines, you can now enable 64k page sizes in the {op-system} kernel on the 64-bit ARM compute machines in your cluster. For more information on setting this parameter, see xref:../post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-compute-managing.adoc#multi-architecture-enabling-64k-pages_multi-architecture-compute-managing[Enabling 64k pages on the Red Hat Enterprise Linux CoreOS (RHCOS) kernel].
```

Comment:
I think this can be directly replaced with an attribute:
```suggestion
On {product-title} {product-version} clusters with multi-architecture compute machines, you can now enable 64k page sizes in the {op-system} kernel on the 64-bit ARM compute machines in your cluster. For more information on setting this parameter, see xref:../post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-compute-managing.adoc#multi-architecture-enabling-64k-pages_multi-architecture-compute-managing[Enabling 64k pages on the {op-system-first} kernel].
```

Reply:
That should work yeah, lemme check !